api.ts

```diff
@@ -1176,9 +1193,41 @@ const MaxPermissionsPerRow = 1000;
 export interface GetPermissionsResult {
   /**
    * The access definition for each row that was passed to {@link executeGetPermissions}.
-   *
    */
   rowAccessDefinitions: RowAccessDefinition[];
+
+  /**
+   * NOTE: I think we need this continuation no matter what we do about the rest of this proposal.
+   */
+  continuation?: Continuation;
+}
+
+export interface GetIncrementalPermissionsResult {
+  /**
+   * If the API you're calling gives you a list of changes to permissions, you can use this.
+   * You cannot use both this and {@link rowAccessDefinitions} in the same result.
+   */
+  incrementalRowAccessDefinitions?: IncrementalRowAccessDefinition[];
+
+  /**
+   * If the API you're using makes you look up the entire set of permissions for an item, use
+   * this field. You cannot use both this and {@link incrementalRowAccessDefinitions} in the same result.
+   */
+  rowAccessDefinitions?: RowAccessDefinition[];
+
+  /**
+   * If provided, we'll pass this into the next invocation. This enables paginating through long lists
+   * of permissions.
+   */
+  continuation?: Continuation;
+
+  /**
+   * This value will be provided to {@link executeGetIncrementalPermissions} the next time Coda crawls permissions.
+   *
+   * Coda assumes that if you return this, your {@link executeGetIncrementalPermissions} can be called on a
+   * different (potentially more frequent) cadence than incremental data syncs.
+   */
+  incrementalPermissionsContinuation?: Continuation;
```

Comment:
We call this `Completion` in the normal execute path I think?  Would be good to be consistent on naming.

Reply:
`completion` is an instance of `SyncCompletionMetadata` which has both `incrementalContinuation` and `incrementalPermissionsContinuation` inside of it. I worried that would be confusing to figure out which one should be populated in this instance. cc @ekoleda-codaio